app-header-layout/test/app-header-layout.html

```diff
@@ -127,23 +116,18 @@
         headerLayout = fixture('trivialFixedHeader');
       });
 
-      test('Issue #406', function(done) {
-        var header = document.createElement('app-header');
-        header.innerHTML = '<app-toolbar></app-toolbar>'
-        header.slot = 'header';
-        header.fixed = true;
-        headerLayout.appendChild(header);
-
-        Polymer.dom.flush();
-
-        requestAnimationFrame(function() {
-          requestAnimationFrame(function() {
-            assert.equal(headerLayout.$.contentContainer.style.marginTop,
-                header.offsetHeight + 'px');
-            done();
-          });
+      test('Issue #406', function() {
+        flush(function() {
+          var header = document.createElement('app-header');
+          header.innerHTML = '<app-toolbar></app-toolbar>'
+          header.slot = 'header';
+          header.fixed = true;
+          Polymer.dom(headerLayout).appendChild(header);
+          Polymer.dom.flush();
+          assert.equal(headerLayout.$.contentContainer.style.marginTop,
+              header.offsetHeight + 'px');
+           });
```

Comment:
This changes technically not related to this PR but instead is to fixup the failing tests from the [previous PR](https://github.com/PolymerElements/app-layout/pull/416).  Do you mind doing a separate PR so we can get this merge sooner?  So we can have the tests passing in travis again.  Thanks!